build.gradle

```diff
@@ -18,55 +18,35 @@ import java.text.SimpleDateFormat
  */
 
 buildscript {
-	ext.kotlinVersion = '1.3.41'
 	repositories {
 		mavenCentral()
 		maven { url "https://repo.spring.io/plugins-release" }
 	}
-	dependencies {
-		classpath 'org.springframework.build.gradle:propdeps-plugin:0.0.7'
```

Comment:
note: propdeps has been removed in this PR, which means optional dependencies won't appear in the pom anymore (like it has been the case in reactor-core for a long time now)